### PR TITLE
Change rejected routes error message to verbose logging

### PR DESCRIPTION
**Pull request by imcsk8**
Instead of printing an error message when a route is invalid,
print a log error with verbose loglevel.

Fixes: rhbz#1491717

### CODE REVIEW

File: pkg/router/controller/host_admitter.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
 
+	"github.com/golang/glog"
```
Comment (Miciah) [inline pkg/router/controller/host_admitter.go]: This is already imported on line 6.

File: pkg/router/controller/host_admitter.go [hunk 2]
```diff
@@ -4,12 +4,11 @@ import (
...
 	"github.com/golang/glog"
+	routeapi "github.com/openshift/origin/pkg/route/apis/route"
+	"github.com/openshift/origin/pkg/router"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/watch"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
-
-	routeapi "github.com/openshift/origin/pkg/route/apis/route"
-	"github.com/openshift/origin/pkg/router"
 )
```
Comment (pravisankar) [inline pkg/router/controller/host_admitter.go]: Why these lines are moved? This is not related to this change and also I don't think we want to do that. This is what we follow in other places:
```
import (
standard imports
+ blank
k8s imports
+ blank
openshift imports
+ blank
Other imports
)
Except standard imports, other imports may be in any order.
```
